src/helix-ui/elements/HXErrorElement.js

```diff
@@ -1,21 +1,13 @@
 import { HXElement } from './HXElement';
 import shadowStyles from './HXErrorElement.less';
-
-const tagName = 'hx-error';
-const template = document.createElement('template');
-
-template.innerHTML = `
-  <style>${shadowStyles}</style>
-  <hx-icon type="exclamation-circle"></hx-icon>
-  <slot></slot>
-`;
```

Comment:
This original implementation resulted in additional whitespace between the `<hx-icon>` and the `<slot>` that the browser misinterpreted as breaking whitespace (also breaking expected appearance when applying CSS).

`\n  <style>' + shadowStyles$3 + '</style>\n  <hx-icon type="exclamation-circle"></hx-icon>\n  <slot></slot>\n`